Re-import WPT css/css-fonts

vitorroriz opened the pull request:
#### d1e5c8f55bcf50fbe8d09dca25e3f07b9acbdc3e
<pre>
Re-import WPT css/css-fonts
<a href="https://bugs.webkit.org/show_bug.cgi?id=294399">https://bugs.webkit.org/show_bug.cgi?id=294399</a>
<a href="https://rdar.apple.com/153218058">rdar://153218058</a>

Reviewed by Matthieu Dubet.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/WEB_FEATURES.yml: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/WEB_FEATURES.yml.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-palette-animation-not-specified-endpoints-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-palette-animation-not-specified-endpoints.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-palette-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-palette-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/multiple-elements-font-palette-animation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/multiple-elements-font-palette-animation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/calc-in-font-variation-settings-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/calc-in-font-variation-settings.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/discrete-no-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/discrete-no-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-in-iframe-print-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-print-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-scoped-to-document-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-scoped-to-document-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-scoped-to-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-local-not-family-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-local-not-family-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-local-not-family.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-range-order.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-sign-function-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-style-normal-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-style-normal.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-family-name-000-expected.xht: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-family-name-000.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-family-name-000.xht:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-family-name-024.xht:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-23b-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-non-ident-font-family-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-23b.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-36.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-non-ident-font-family-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-non-ident-font-family-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-relative-color-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-shorthand-serialization-font-stretch-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-shorthand-serialization-font-stretch.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-009-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-009-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-010-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-010.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-012.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-013.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-generic-font-fallback-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-ic-height-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-ic-height-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-ic-height.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-interpolation-math-functions-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-interpolation-math-functions.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-metrics-override-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-metrics-override-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-metrics-override.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-nan-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-text-orientation-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-text-orientation-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-text-orientation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-stretch-interpolation-math-functions-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-stretch-interpolation-math-functions.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-style-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-style-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-08-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-08-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-08.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-position-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-position-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-position-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-style-oblique-only-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-style-oblique-only-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-style-oblique-only.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-caps-invalidation-container-sizing-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-caps-invalidation-container-sizing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-04-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-04-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-04.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-05-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-05-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-05.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variation-settings-calc-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variation-settings-calc.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-weight-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-weight-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/format-specifiers-variations-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/format-specifiers-variations.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-003-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/idlharness-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/idlharness.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/inheritance-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/inheritance.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/italic-oblique-fallback-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/italic-oblique-fallback-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/italic-oblique-fallback.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/language-specific-01.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/font-unicode-presented-as-emoji-outline-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/font-unicode-presented-as-emoji-outline-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/font-unicode-presented-as-emoji-outline.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/resources/AhemCrossmarkSupport.otf: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-001.tentative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-002.tentative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/palette-mix-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/palette-mix-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-size-adjust-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-alias-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-alias.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-style-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-style-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-style-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-style-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-style-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-style-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-style-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-style-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-computed-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-computed-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-invalid-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-invalid-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-valid-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-valid-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/separator-test-font.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/MPLUS1-Regular_without-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoColorEmoji-Regular_subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoEmoji-Regular_subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoEmoji-Regular_without-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoSansJP-Regular_with-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoSansMath-Regular_without-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/STIXTwoMath-Regular_with-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/separators-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/separators-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/separators.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-tentative-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-unicode-range-system-fallback-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-unicode-range-system-fallback-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-unicode-range-system-fallback.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust.tentative-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/css/variation-sequences.css: Added.
(@font-face):
(body):
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/css/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/font-family-keywords.js:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/iframe-missing-font-face-rule.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/iframe-using-ahem-as-web-font.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/iframe-without-web-font.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/js/variation-sequences.js: Added.
(const.range):
(const.permutations):
(getMatchedFamilyForVariationSequence):
(generateContent):
(generateVariationSequenceTests):
(generateVariationSequenceRefs):
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/js/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/synthetic-bold-space-width-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/synthetic-bold-space-width-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/synthetic-bold-space-width-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/synthetic-bold-space-width.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-2-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-2-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-4-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-4-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-4.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test_datafont_same_origin-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test_font_feature_values_parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test_font_feature_values_parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variable-in-feature-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variable-in-font-variation-settings-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variable-in-font-variation-settings.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variation-sequences-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variation-sequences-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variation-sequences.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-descriptors-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-descriptors.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-font-matching-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-font-matching.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/font-style-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/font-style-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/font-weight-matching.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/slnt-backslant-variable.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/slnt-variable.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/web-font-no-longer-accessible-when-stylesheet-removed-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/web-font-no-longer-accessible-when-stylesheet-removed-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/web-font-no-longer-accessible-when-stylesheet-removed.html: Added.
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/296179@main">https://commits.webkit.org/296179@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/82a1f3a5fb5232673009b13e77b0272b50ddbc1f

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/107578 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/27261 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/17671 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/112795 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/58120 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/27952 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/35763 "Built successfully") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/81675 "Found 4 new test failures: imported/w3c/web-platform-tests/feature-policy/feature-policy-frame-policy-allowed-for-self.https.sub.html imported/w3c/web-platform-tests/feature-policy/feature-policy-frame-policy-timing.https.sub.html imported/w3c/web-platform-tests/fetch/api/response/response-cancel-stream.any.serviceworker.html imported/w3c/web-platform-tests/fetch/api/response/response-cancel-stream.any.sharedworker.html (failure)") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/110507 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/22155 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/96962 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/62055 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/21592 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/15101 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/57559 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/91525 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/15135 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/115896 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/34646 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/25551 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/90708 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/35022 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/93212 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/90449 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/23072 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/35368 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/13151 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/30409 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/34567 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/34313 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/37673 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/35976 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

mdubet commented on LayoutTests/TestExpectations:
```diff
@@ -4517,7 +4517,7 @@ webkit.org/b/203606 imported/w3c/web-platform-tests/css/css-display/display-cont
 
 # Untriaged css-fonts failures
 webkit.org/b/206881 imported/w3c/web-platform-tests/css/css-fonts/first-available-font-001.html [ ImageOnlyFailure ]
-webkit.org/b/206881 imported/w3c/web-platform-tests/css/css-fonts/first-available-font-002.html [ ImageOnlyFailure ]
```
We should mark it as [ Skip ] with your comment instead of commenting the full line

vitorroriz committed:
Re-import WPT css/css-fonts
https://bugs.webkit.org/show_bug.cgi?id=294399
rdar://153218058

Reviewed by Matthieu Dubet.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/WEB_FEATURES.yml: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/WEB_FEATURES.yml.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-palette-animation-not-specified-endpoints-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-palette-animation-not-specified-endpoints.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-palette-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-palette-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/multiple-elements-font-palette-animation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/multiple-elements-font-palette-animation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/calc-in-font-variation-settings-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/calc-in-font-variation-settings.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/discrete-no-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/discrete-no-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-in-iframe-print-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-print-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-scoped-to-document-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-scoped-to-document-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/downloadable-font-scoped-to-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-local-not-family-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-local-not-family-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-local-not-family.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-range-order.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-sign-function-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-style-normal-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-face-style-normal.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-family-name-000-expected.xht: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-family-name-000.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-family-name-000.xht:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-family-name-024.xht:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-23b-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-non-ident-font-family-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-23b.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-36.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-non-ident-font-family-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-non-ident-font-family-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-palette-relative-color-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-shorthand-serialization-font-stretch-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-shorthand-serialization-font-stretch.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-009-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-009-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-010-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-010.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-012.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-013.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-generic-font-fallback-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-ic-height-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-ic-height-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-ic-height.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-interpolation-math-functions-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-interpolation-math-functions.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-metrics-override-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-metrics-override-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-metrics-override.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-nan-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-text-orientation-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-text-orientation-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-text-orientation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-stretch-interpolation-math-functions-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-stretch-interpolation-math-functions.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-style-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-style-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-08-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-08-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-08.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-position-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-position-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-position-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-style-oblique-only-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-style-oblique-only-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-synthesis-style-oblique-only.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-caps-invalidation-container-sizing-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-caps-invalidation-container-sizing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-emoji-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-04-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-04-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-04.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-05-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-05-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variant-position-05.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variation-settings-calc-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-variation-settings-calc.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-weight-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-weight-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/format-specifiers-variations-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/format-specifiers-variations.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-003-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/generic-family-keywords-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/idlharness-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/idlharness.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/inheritance-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/inheritance.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/italic-oblique-fallback-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/italic-oblique-fallback-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/italic-oblique-fallback.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/language-specific-01.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/font-unicode-presented-as-emoji-outline-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/font-unicode-presented-as-emoji-outline-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/font-unicode-presented-as-emoji-outline.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/resources/AhemCrossmarkSupport.otf: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-001.tentative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-002.tentative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/font-size-math-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/math-script-level-and-math-style/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/palette-mix-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/palette-mix-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-size-adjust-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-alias-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-alias.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-style-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-style-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-style-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-style-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-style-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-style-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-style-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-style-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-synthesis-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-computed-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-computed-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-invalid-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-invalid-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-valid-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-stretch-valid-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-width-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/separator-test-font.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/MPLUS1-Regular_without-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoColorEmoji-Regular_subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoEmoji-Regular_subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoEmoji-Regular_without-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoSansJP-Regular_with-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/NotoSansMath-Regular_without-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/STIXTwoMath-Regular_with-cmap14-subset.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/vs/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/separators-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/separators-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/separators.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-tentative-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-unicode-range-system-fallback-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-unicode-range-system-fallback-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-unicode-range-system-fallback.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust.tentative-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/css/variation-sequences.css: Added.
(@font-face):
(body):
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/css/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/font-family-keywords.js:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/iframe-missing-font-face-rule.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/iframe-using-ahem-as-web-font.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/iframe-without-web-font.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/js/variation-sequences.js: Added.
(const.range):
(const.permutations):
(getMatchedFamilyForVariationSequence):
(generateContent):
(generateVariationSequenceTests):
(generateVariationSequenceRefs):
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/js/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/synthetic-bold-space-width-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/synthetic-bold-space-width-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/synthetic-bold-space-width-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/synthetic-bold-space-width.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-2-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-2-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-bold.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-4-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-4-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-4.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test-synthetic-italic.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test_datafont_same_origin-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test_font_feature_values_parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test_font_feature_values_parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variable-in-feature-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variable-in-font-variation-settings-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variable-in-font-variation-settings.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variation-sequences-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variation-sequences-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variation-sequences.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-descriptors-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-descriptors.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-font-matching-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-font-matching.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/font-style-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/font-style-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/font-weight-matching.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/slnt-backslant-variable.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/slnt-variable.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/web-font-no-longer-accessible-when-stylesheet-removed-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/web-font-no-longer-accessible-when-stylesheet-removed-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/web-font-no-longer-accessible-when-stylesheet-removed.html: Added.
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:

Canonical link: https://commits.webkit.org/296179@main
